subsys/bluetooth/host/hci_core.c

```diff
@@ -3814,6 +3814,11 @@ static const struct event_handler normal_events[] = {
 		      hci_encrypt_key_refresh_complete,
 		      sizeof(struct bt_hci_evt_encrypt_key_refresh_complete)),
 #endif /* CONFIG_BT_SMP || CONFIG_BT_BREDR */
+#if defined(CONFIG_BT_REMOTE_VERSION)
+	EVENT_HANDLER(BT_HCI_EVT_REMOTE_VERSION_INFO,
+		      bt_hci_evt_read_remote_version_complete,
+		      sizeof(struct bt_hci_evt_remote_version_info)),
+#endif /* CONFIG_BT_REMOTE_VERSION */
```

Comment:
It seems that you haven't introduced the  `bt_hci_evt_read_remote_version_complete()` yet in this patch - it only comes in a later one. Btw, a good way to check that each commit builds on its own is something like `git rebase --exec="west build" origin/master`. That will run the command given to the `--exec` switch for every commit that's part of the rebase.

Reply:
Actually, you haven't introduced the `CONFIG_BT_REMOTE_VERSION` option either here. I know this is a rather pedantic approach but it helps later when doing something like git bisect since every commit that it might land on should be sane and buildable.

Reply:
Thanks, good catch. I don't know exactly how it happened but some of my fixup commits ended up in the wrong commits.
The  `--exec=""` was a good tip. Although in this case it wouldn't have caught it as it still compiled due to the CONFIG_BT_REMOTE_VERSION not having been defined.

Moved the change to the right commit.